Sources/App/routes.swift

```diff
@@ -61,7 +61,10 @@ func routes(_ app: Application) throws {
             app.get(":owner", ":repository", ":reference", "documentation") {
                 try await packageController.documentation(req: $0, fragment: .documentation)
             }
-            app.get(":owner", ":repository", ":reference", "documentation", "**") {
```

Comment:
I wonder if we're missing any routes if we drop this, although I may have added this for sake of completeness more than anything.

Reply:
No. This one is important if people land deeper into the documentation.

Eg. `/owner/repo/ref/documentation/symbol/method/something_else`

Reply:
Well, should we be removing it then? :) This is getting deleted in this PR!

Reply:
Deleted?

Reply:
Yes, this route as is is being removed isn't it?

<img width="667" alt="Screenshot 2022-07-29 at 17 55 21" src="https://user-images.githubusercontent.com/65520/181797941-d3bcecec-286a-441c-8424-59101ddd1213.png">

Reply:
(the others might be 1:1 replacements but I'm not 100% sure if they are, i.e. how the `**` actually captures and deals with precedence etc)

Reply:
Ah, I see. Not really. I needed to extract the first part of the fragment out into a named parameter `:archive`, so this route is now basically the one on L67.